addons/mail/models/mail_thread.py

```diff
@@ -2438,6 +2438,8 @@ def _notify_thread_by_email(self, message, recipients_data, msg_vals=False,
         :param send_after_commit: if force_send, tells whether to send emails after
           the transaction has been committed using a post-commit hook;
         """
+        if self.env.context.get('mail_auto_subscribe_no_notify'):
+            return
```

Comment:
Hello @RaoufGhrissi 👋🏼 
Using a context in mail will never be merged.
If you could explain the need, @tde-banana-odoo might help once he has time (which is not the case right now as they are rushing on knowledge, and this is not an issue for this task as it is not a priority).

I haven't reviewed the rest though as it is really a blocking point. I'll wait for this to be solved.

Thanks for your work 🙂

Reply:
Hello, the main need is to not send emails to subscribers when we copy the task 
so i added the context when copy is called
https://github.com/odoo/odoo/pull/91174/files#:~:text=.id)-,%40api.returns(%27self%27%2C%20lambda%20value%3A,self%20%3D%20self.with_context(mail_auto_subscribe_no_notify%3DTrue),-project%20%3D%20super

question: why it can be merged ?
i saw we use something similar in crm 
![image](https://user-images.githubusercontent.com/64937092/191205239-332a2833-9ccf-4764-8161-6ee1bf8f0a44.png)

Reply:
Oh, ok. But I am unsure that adding more will make @tde-banana-odoo happy 🙂 
There is maybe another way to meet the requirements. But in the same time, when we copy a task, it creates a record, so if there are followers it seems legit to me that they get notified. I don't really understand the need.
Could you elaborate?

Reply:
the only detail i have i the task description:
- prevent the sending of emails (assignment notifications, emails on stages) when duplicating a project or a task.

if you need more details, we can discuss with AJU on discord. what do you think ?

Reply:
Hello @RaoufGhrissi 👋🏼 
Can't we use `mail_create_nolog`, `mail_create_nosubscribe` or `mail_notrack`? Or a combination of them?
```python
DISABLED_MAIL_CONTEXT = {
    'tracking_disable': True,
    'mail_create_nolog': True,
    'mail_create_nosubscribe': True,
    'mail_notrack': True,
    'no_reset_password': True,
}
```

Reply:
Hello @IT-Ideas i think we can use `mail_create_nosubscribe`, i finish the bug i have in hand and move to fix it, do you want to finish yout review and fix all comments at the same time or it's the only comment you have?

Reply:
Hello @RaoufGhrissi I am not done with the review yet as it was a blocking issue for me.
You can already test push that change before I proceed.
Thanks 🙂

Reply:
Hello @RaoufGhrissi did you give it a try?

Reply:
oh my bad @IT-Ideas  my comment was pending from a long period 
![image](https://user-images.githubusercontent.com/64937092/197571727-f5193a37-d631-419f-b635-dd6f18a45e4b.png)

Reply:
hey @IT-Ideas, in case you didn't see my mention as Alexandra pinged my about this task

Reply:
Yes @aju-odoo asked to focus on bug fixes up to now. So I don't know why she pushes you with this...
This context thing is problematic to me. I am not sure how to handle the issue and if we should... Maybe @tde-banana-odoo could have a hint.

cc @xavierbol

Reply:
thanks @IT-Ideas , maybe she though that i'm blocked and wanted to help me. No worries. Good luck for what you're doing